src/main/java/stsjorbsmod/powers/PossessionPower.java

```diff
@@ -0,0 +1,63 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.JorbsMod;
+
+public class PossessionPower extends CustomJorbsModPower{
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(PossessionPower.class);
+    public static final String POWER_ID = STATIC.ID;
+    private final AbstractMonster target;
+
+    public PossessionPower(AbstractMonster owner, int amount) {
+        super(STATIC);
+        this.type = PowerType.DEBUFF;
+        this.owner = owner;
+        this.amount = amount;
+        this.target = owner;
+        this.isTurnBased = true;
+
+        updateDescription();
+    }
+
+    @Override
+    public void atStartOfTurn() {
+        AbstractMonster m = this.target;
+        JorbsMod.logger.info("m and this.owner: " + m);
```

Comment:
I think the logging statements can probably be removed after you're comfortable the power is working correctly; no need to spam the log with them if we aren't actively debugging